ambry-account/src/test/java/com/github/ambry/account/AccountTestUtils.java

```diff
@@ -123,10 +123,9 @@ static void generateRefAccounts(Map<Short, Account> idToRefAccountMap,
             random.nextBoolean() ? Container.ContainerStatus.ACTIVE : Container.ContainerStatus.INACTIVE;
         String containerDescription = UUID.randomUUID().toString();
         boolean containerCaching = random.nextBoolean();
-        // TODO make these randomly generated once V2 container writes are enabled
```

Comment:
This can be changed since the Container will use the default values internally if the json version does not support a certain field.